tooling/minimize/tests/pass/tree_borrows/sb_fails.rs

```diff
@@ -0,0 +1,65 @@
+//@ compile-flags: --minimize-tree-borrows
+
+// The tests were taken from Miri Tree Borrows 
+// https://github.com/rust-lang/miri/blob/6680b2fa09496b60c40c6ce09449f46efbf253d5/tests/pass/tree_borrows/sb_fails.rs
+// FIXME: Some tests in the original test suite are not currently supported by MiniRust 
```

Comment:
Can you please add a list of which tests, so that we can add them later? This applies to everywhere that you added this FIXME.